packages/opentelemetry-node/src/instrumentation/PluginLoader.ts

```diff
@@ -83,6 +83,21 @@ export class PluginLoader {
         return this;
       }
 
+      const alreadyRequiredModules = Object.keys(require.cache);
+      const requiredModulesToHook = modulesToHook.filter(
+        name =>
+          alreadyRequiredModules.find(
+            cached => require.resolve(name) === cached
```

Comment:
`require.resolve` will throw when `name` is not found. This needs to be wrapped in a try/catch.

Reply:
Indeed, i fixed it